services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/RajaArvoLuokittelija.kt

```diff
@@ -22,16 +22,14 @@ object RajaArvoLuokittelija {
 
     fun linjaautoliikenteenRuuhkavuoroluokka(countOfRushHourBuses: Int) =
         when {
-            countOfRushHourBuses >= 21 ->
+            countOfRushHourBuses > 20 ->
```

Comment:
Doesn't this need editing as well?

https://github.com/City-of-Helsinki/haitaton-backend/blob/3a938e130d9fc21c90fc68b24f135fa7175b2efd/services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluLaskentaService.kt#L130-L131

Reply:
You are right! I missed this. Removed the unnecessary if-statement.